openmetadata-ui/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx

```diff
@@ -252,6 +252,17 @@ const Ingestion: React.FC<IngestionProps> = ({
     }
   };
 
+  const isDataSightIngestionExists = useMemo(
+    () =>
+      !isEmpty(
+        !isEmpty(ingestionData) &&
+          ingestionData.filter(
+            (ingestion) => ingestion.pipelineType === PipelineType.DataInsight
+          )
```

Comment:
https://lodash.com/docs/4.17.15#some
This is what you are looking for and this will probably reduce complexity over here

Reply:
Thanks, Updated the code.